src/main/java/blang/distributions/LogUniform.bl

```diff
@@ -0,0 +1,33 @@
+package blang.distributions
+
+model LogUniform {
+  random RealVar realization
+  
+  /** The left end point \(m\) of the interval. \(m \in (\infty, M)\) */
+  param  RealVar min
+  
+  /** The right end point of the interval. \(M \in (m, \infty)\) */
+  param  RealVar max
+  
+  param RealVar base
+  
+  laws {
+    logf(min, max) {
+      if (max - min <= 0.0) return NEGATIVE_INFINITY
+      return - log(max - min)
+    }
+    logf(realization, min, max, base) {
+      if (base <= 0) return NEGATIVE_INFINITY
+      val logr = log(realization)
+      val logb = log(base)
+      val transformed = logr / logb
+      if (min <= transformed && transformed <= max) return -logr - log(logb)
```

Comment:
Might be more robust to have a realization > 0 check before taking log(realization) (?)

Reply:
Thanks! Just pushed these fixes.